conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -71,6 +71,7 @@ dependencies:
 - rmm==25.12.*,>=0.0.0a0
 - scikit-build-core>=0.10.0
 - scikit-learn>=1.4
```

Comment:
Seems like this is duplicated. Is there another pinning that needs an upper bound?